interface/forms/dictation/view.php

```diff
@@ -12,6 +12,8 @@
 
 use OpenEMR\Core\Header;
 include_once("../../globals.php");
+require_once("../../../public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
```

Comment:
Since it's autoloaded, no include/require is needed. The class is magically available in all scripts. However, will need to rebase your code on the most current codebase (in order to bring this code in). Also should delete entire public/assets/htmlpurifier-4.9.2 directory from your branch.